src/main/resources/db/changelog/changes/07-create-books-categories-table.yaml

```diff
@@ -0,0 +1,24 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books_categories-table
+      author: anastasiiakulinich
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                    foreignKeyName: fk_book
+                    references: books(id)
+              - column:
+                  name: category_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                    foreignKeyName: fk_category
+                    references: categories(id)
```

Comment:
add empty line at EOF where you see this
![image](https://github.com/user-attachments/assets/5cdb6d50-48e7-4c24-b327-0b823aec5392)